reactivity-components.Rmd

```diff
@@ -128,7 +128,7 @@ The final important property of reactive expressions is that they _cache_ their
 
 The first time a reactive expression is called, it will execute its code body, and depending on what that code does, it might take a significant amount of time. But when the calculation is complete, the resulting value will be both returned to the caller _and_ remembered by the reactive expression. Subsequent calls to the reactive expression take essentially no time at all, as the saved value can be returned instantly. If a reactive expression depends on reactive values or expressions, then any changes to those will cause the cached value to be discarded. When that happens, the next call to the reactive expression will again cause an actual calculation, whose result will then be saved for subsequent calls.
 
-These particular properties--laziness, caching, reactivity, and lack of side effects--combine to give us an elegant and versatile building block for reactive programming.
+These particular properties -- laziness, caching, reactivity, and lack of side effects -- combine to give us an elegant and versatile building block for reactive programming.
```

Comment:
FWIW I think both before and after are incorrect it should be (e.g.) `start of sentence---parenthetical---end of sentence`, but I haven't rewritten these chapters yet, so I'll get to it when I do.

Reply:
True. I enjoyed reading the Work In Progress anyway 😉
Glad I could contribute in this form!